src/read-only-handler.ts

```diff
@@ -39,7 +39,10 @@ export class ReadOnlyHandler extends BaseProxyHandler {
     set(shadowTarget: ReactiveMembraneShadowTarget, key: PropertyKey, value: any): boolean {
         if (process.env.NODE_ENV !== 'production') {
             const { originalTarget } = this;
-            throw new Error(`Invalid mutation: Cannot set "${key.toString()}" on "${originalTarget}". "${originalTarget}" is read-only.`);
+            const msg = Array.isArray(originalTarget) ?
```

Comment:
You can import `isArray` from the shared module:

https://github.com/salesforce/observable-membrane/blob/5e64a21f4b18a0835515b326fd5aa1ef0cac9baf/src/shared.ts#L26